sda/cmd/verify/verify.go

```diff
@@ -392,6 +392,15 @@ func main() {
 					continue
 				}
 
+				if err := db.UpdateFileEventLog(message.FileID, "verified", delivered.CorrelationId, "ingest", "{}", string(verifiedMessage)); err != nil {
```

Comment:
why is this added..?

Reply:
Because `setVerified` is moved to GO and to keep all calls identical in `s3inbox`, `ingest` and `verify`

This change is brought by the fact that ingest now can handle cases correctly, re-ingesting a file where a new version is uploaded to the inbox, with new checksum.